tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -347,6 +351,10 @@ these properties, and if you don't write to them, Godot will optimize away the c
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
 | out vec2 **ALPHA_TEXTURE_COORDINATE**  |                                                                                                  |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
+| out float **PREMUL_ALPHA_FACTOR**      | Premultiplied alpha factor. Only effective if ``render_mode blend_premul_alpha;`` is used.       |
+|                                        | This should be written to when using a *shaded* material with premultiplied alpha blending for   |
+|                                        | interaction with lighting. This is not required for unshaded materials.                          |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
```

Comment:
Any ideas of a concrete example? I've based this description on the existing class reference, but it's not really obvious how you're supposed to write to it in a typical shaded PBR material scenario.

Reply:
cc @clayjohn @Chaosus